Atomic/trust.py

```diff
@@ -167,6 +167,11 @@ def modify_default(self):
                 policy["default"][0]["type"] = default_type_map[self.args.default_policy]
             else:
                 policy["default"] = [ { "type": default_type_map[self.args.default_policy] }]
+            if default_type_map[self.args.default_policy] == "reject":
+                if not "transports" in policy:
+                    policy["transports"] = {}
+                # we don't want to reject push from docker-daemon
+                policy["transports"]["docker-daemon"] = { "": [{"type":"insecureAcceptAnything"}] }
```

Comment:
This would override whatever the user added for this.  I think we should just make this the default and put it in our policy.conf file. Then if the user changes the default, it will stay with whatever the user changed it to.